docs/user_guide/1_4_Installing_the_software.md

```diff
@@ -0,0 +1,113 @@
+
+# Starting the driver
+
+Launch the driver at a terminal (in the container), typing:
+
+```eval_rst
+.. prompt:: bash $
+
+    roslaunch sr_ethercat_hand_config sr_system.launch
+```
+
+## Lights in the hand
+When the ROS driver is running you should see the following lights on the Palm:
+
+```eval_rst
+========================   =============       ================    =================================
+Light                      Colour              Activity            Meaning
+========================   =============       ================    =================================
+Run                        Green               On                  Hand is in Operational state
+CAN1/2 Transmit            Blue                V.fast flicker      Demand values are being sent to the motors
+CAN1/2 Receive             Blue                V.fast flicker      Motors are sending sensor data
+Joint sensor chip select   Yellow              On                  Sensors being sampled
+========================   =============       ================    =================================
+```
+
+After killing the driver, the lights will be in a new state:
+```eval_rst
+========================   =============       ================    =================================
+Light                      Colour              Activity            Meaning
+========================   =============       ================    =================================
+Run                        Green               Blinking            Hand is in Pre-Operational state
+CAN1/2 Transmit            Blue                Off                 No messages transmitted on CAN 1/2
+CAN1/2 Receive             Blue                Off                 No messages received on CAN 1/2
+Joint sensor chip select   Yellow              Off                 Sensors not being sampled
+========================   =============       ================    =================================
+```
+
+# Saving log files and uploading data to our server
+When running the one-liner, you will also notice a second icon named `Save logs` that is used to retrieve and copy all the available logs files from the active containers locally on your Desktop. This icon will create a folder that matches the active container's name and the next level will include the date and timestamp it was executed. When it starts, it will prompt you if you want to continue, as by pressing yes it will close all active containers. After pressing "yes", you will have to enter a description of the logging event and will start coping the bag files, logs and configuration files from the container and then exit. Otherwise, the window will close and no further action will happen. If you provided an upload key with the one-liner installation then the script will also upload your LOGS in compressed format to our server and notify the Shadow's software team about the upload. This will allow the team to fully investigate your issue and provide support where needed.
```

Comment:
```suggestion
After running the one-liner, you will also notice a second icon named `Save logs` that is used to retrieve and copy all the available logs files from the active containers locally to your Desktop. This icon will create a folder that matches the active container's name and the next level will include the date and timestamp it was executed. When it starts, it will prompt you if you want to continue, if you press yes it will close all active containers. After pressing "yes", you will have to enter a description of the logging event and will start coping the bag files, logs and configuration files from the container and then exit. Otherwise, the window will close and no further action will happen. If you provided an upload key with the one-liner installation then the script will also upload your LOGS in compressed format to our server and notify the Shadow's software team about the upload. This will allow the team to fully investigate your issue and provide support where needed.
```